src/TimeKeeper.php

```diff
@@ -31,29 +34,50 @@ class TimeKeeper
     public function __construct(DateTime $gameEpoch, int $gameOffsetSeconds, int $gameDaysPerDay)
     {
         $gameEpochCopy = clone($gameEpoch);
-        $this->adjustedEpoch = $gameEpochCopy->add(
-            $this->interval(0, 0, 0, 0, $gameOffsetSeconds)
-        );
+
+        if ($gameOffsetSeconds < 0) {
```

Comment:
User's expectation: If new day happens to be at 1 o'clock, I should set it of by -1 hour instead of +5 hours.

So far this has been "undefined" behaviour: It has not been checked and thus resulted in a parse error from \DateTimeInterval due to the string being "PT-1S" which is not valid.